### PR TITLE
Remove supporting announcements code

### PR DESCRIPTION
**Pull request by gclssvglx**
## What

Remove remaining code used to render announcements.

## Why

On [1st April - Remove NHS Box, Announcements, Timeline Entries, Statistics sections from landing page and move DA Links](https://trello.com/c/zxAAM52Y) we stopped rendering the announcements on the landing page. Now we need to remove the rest of the frontend code for announcements.

[Trello](https://trello.com/c/YRPa1vrE/811-remove-supporting-announcements-section-code-from-collections)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/presenters/coronavirus_landing_page_presenter.rb [hunk 1]
```diff
@@ -63,9 +61,7 @@ def timelines_for_nation
 private
 
   def build_faq_main_entity(content_item)
-    question_and_answers = []
-    question_and_answers.push build_announcements_schema(content_item)
-    question_and_answers.concat build_sections_schema(content_item)
+    build_sections_schema(content_item)
```
[the remark this answers is not in the document]
Comment (leenagupte) [reply app/presenters/coronavirus_landing_page_presenter.rb:64]: Not a blocker: `build_faq_main_entity` is only being used in one place and could probably be replaced by calling `build_sections_schema` directly.

File: app/views/coronavirus_landing_page/components/landing_page/_announcements_section.html.erb [hunk 2]
```diff
@@ -1,29 +0,0 @@
...
-          data-module="gem-track-click"
-          data-track-category="pageElementInteraction"
-          data-track-action="Announcements"
-          data-track-label="<%= details.see_all_announcements_link["href"] %>"
-        >
-          <%= details.see_all_announcements_link["text"] %>
-        </a>
-      </p>
-    <% end %>
-    <%= render partial: 'coronavirus_landing_page/components/landing_page/risk_level_section', locals: { data: details.risk_level } if details.risk_level %>
```
Comment (leenagupte) [inline app/views/coronavirus_landing_page/components/landing_page/_announcements_section.html.erb:27]: I think this partial, `_risk_level_section.html.erb` is only rendered here, so should be removed as well.

File: app/presenters/coronavirus_landing_page_presenter.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (gclssvglx) [reply app/presenters/coronavirus_landing_page_presenter.rb]: Ah yes, that's much nicer - great spot - thanks. Fixed

File: spec/presenters/announcements_presenter_spec.rb [hunk 3]
```diff
@@ -1,12 +0,0 @@
-RSpec.describe AnnouncementsPresenter do
```
[the remark this answers is not in the document]
Comment (gclssvglx) [reply spec/presenters/announcements_presenter_spec.rb]: Whoops! Put back in - thanks

File: spec/presenters/coronavirus_landing_page_presenter_spec.rb [hunk 4]
```diff
@@ -4,9 +4,6 @@
     presenter = described_class.new(coronavirus_landing_page_content_item)
     %i[
       header_section
-      announcements_label
```
[the remark this answers is not in the document]
Comment (gclssvglx) [reply spec/presenters/coronavirus_landing_page_presenter_spec.rb:7]: Moved - thanks

File: app/views/coronavirus_landing_page/components/landing_page/_announcements_section.html.erb (hunk 2 above)
Comment (gclssvglx) [reply app/views/coronavirus_landing_page/components/landing_page/_announcements_section.html.erb:27]: Yeah - quire right - removed

**Comment by gclssvglx** [comment]
> I've given this a red X because a test file has been deleted in error. Also, there's another partial that needs to be deleted.
> 
> Otherwise, this is looking good.

Thanks for the great review @leenagupte 👍  - all fixed up

**Review by leenagupte — COMMENTED** [review]
I think there's only one tiny bit of CSS left to remove.

File: app/views/coronavirus_landing_page/components/landing_page/_risk_level_section.html.erb [hunk 5]
```diff
@@ -1,20 +0,0 @@
-<% if data["show_risk_level_section"] %>
-  <%= render "govuk_publishing_components/components/govspeak", {
-  } do %>
-    <div class="info-notice covid__alert-notice">
```
[the remark this answers is not in the document]
Comment (gclssvglx) [reply app/views/coronavirus_landing_page/components/landing_page/_risk_level_section.html.erb:4]: Nice one - sorry didn't catch that. Thanks fixed

**Review by leenagupte — APPROVED** [review]
Looks good! 💯

**Pull request merged**